components/Participant.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { createRef, useEffect, useState } from 'react';
+
+const Participant = ({ gain, participant, isYou }) => {
+  const [videoRef] = useState(createRef<HTMLVideoElement>());
+
+  const playTrack = track => {
+    if (track.kind === 'audio') {
+      if (!isYou) {
+        // Don't play your own audio
+        var audio = new Audio();
+        audio.srcObject = postProcessAudioTrack(track);
+      }
+    } else if (track.kind === 'video') {
+      taggedLog('Creating new MediaStream');
+      taggedLog('track:', track);
+      const mediaStreamTrack = track.mediaStreamTrack;
+      const stream = new MediaStream();
+      stream.addTrack(mediaStreamTrack);
+      // @ts-ignore
```

Comment:
Ignored because Typescript doesn't know what `videoRef.current` refers to and errors out saying that you can't set `srcObject` on `Unknown`. ¯\_(ツ)_/¯